### ISSUE TITLE
Maintenance group should review all term metadata Definition, Usage, and Notes fields

### ISSUE THREAD

**Issue by baskaufs** [tdwg/ac#115]
I'm opening this issue as an umbrella issue that will encompass several issues previously raised.  

In going through the term list document to make it conform to the Standards Documentation Specification, I was able to sort out some obvious misplacements of examples within definitions or normative material placed in notes.  But there were still many cases where it was questionable whether some text was misplaced.  For example, is it appropriate to refer users to a mutable Wikipedia article from a normative definition?  

At the same time, we should look carefully at our use of borrowed terms.  For the first time, the term list document actually lists the definitions of borrowed terms taken directly from their defining documents.  Those authoritative definitions should be compared to the AC Usage notes for incompatibilities.  For example, the definition says that exif:PixelXDimension "shall not exist in an uncompressed file".  Does that mean that a uncompressed TIFF can't have a value of that property?  There are also cases where the XMP and IPTC specs say that values should be of some weird type.  For example, the value of xmpRights:Owner is supposed to be an "Unordered array of ProperName" and Iptc4xmpExt:CVterm is supposed to be a "bag URI".  What do those types even mean and does it matter if we don't use them?  Refer to Issue #88 (which I'm closing in favor of this one) for more on this subject.

Issue #100 also notes that the IPTC standard has a more recent update.  Do we update our terms to follow the updates of our borrowed vocabularies?  What about the DwC Location terms that are specifically "frozen" at the 9 Dec 2009 version?  Did we really intend to freeze their definitions, or just freeze the list of terms we were borrowing?  

There were also a few cases where term metadata simply weren't clear to me, such as the Usage value for ac:subtype, which says "Conforming applications may choose to ignore these."  What is "these"?

Anyway, rather than accumulating a bunch of individual issues that will clog up the tracker, we need to just do a thorough review of all term metadata as it stands after the rollout of the revised term list document.  If there is any chance that the clarifications that we make may affect existing implementations, then the appropriate change process from [Section 3.2.2 of the Vocabulary Maintenance Specification](https://github.com/tdwg/vocab/blob/master/vms/maintenance-specification.md#32-errata) should be triggered.

**Comment by baskaufs** [comment]
Google doc with proposed changes is [here](https://docs.google.com/document/d/12Ck4t_x9LtG0BgPuBcNNqtYppNCP_Rj5F7bdgkmFvGY/edit?usp=sharing).  Commenting is turned on.

**Comment by baskaufs** [comment]
Blocked by Issues #130, #131, and #132

**Comment by baskaufs** [comment]
All issues blocking this one have been closed.
